lib/mzgl/gl/Graphics.cpp

```diff
@@ -760,7 +445,7 @@ void Graphics::drawRoundedRectShadow(Rectf r, float radius, float shadow) {
 
 #include "DefaultFont.inc.h"
 
-vector<unsigned char> Graphics::getDefaultFontTTFData() {
+std::vector<unsigned char> Graphics::getDefaultFontTTFData() {
```

Comment:
This is what I was alluding to earlier. Each call here will make a memory copy of the data. If you make it a static constexpr array and the compiler can see straight through it and will give you it without the memory over head